config/initializers/_env_var_constants.rb

```diff
@@ -10,7 +10,8 @@
 
```

Comment:
Renaming this file so it loads first in initializers, which then makes `FUND_MAILER_DOMAIN` and `FUND_DOMAIN` available to other initializers (most notably, `devise.rb`).